frontend/src/pages/PostPage/components/ReportModal/index.styles.ts

```diff
@@ -0,0 +1,56 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+export const ReportModalContainer = styled.form`
+  background-color: white;
+  padding: 1em;
+  box-shadow: 0px 1px 7px ${props => props.theme.colors.gray_150};
```

Comment:
```suggestion
  box-shadow: 0px 1px 7px ${props => props.theme.colors.gray_150};
  border-radius: 4px;
```

border-radius가 누락됐어요